http.kafka.cqrs/zilla.json

```diff
@@ -0,0 +1,201 @@
+                                ":authority": "localhost:9090"
+                            }
+                        }
+                    ]
+                },{
+                    "exit": "sse0",
+                    "when":
+                    [
+                        {
+                            "headers":
+                            {
+                                ":path": "/tasks"
+                            }
+                        }
+                    ]
+                }
+            ]
+        },
+        "http_kafka0":
+        {
+            "type" : "http-kafka",
+            "kind": "proxy",
+            "routes":
+            [
+                {
+                    "exit": "kafka0",
+                    "when":
+                    [
+                        {
+                            "method": "GET",
+                            "path": "/tasks/{id};{correlationId}"
+                        },
+                        {
+                            "method": "POST",
+                            "path": "/tasks/{id}"
+                        },
+                        {
+                            "method": "PUT",
+                            "path": "/tasks/{id}"
+                        },
+                        {
+                            "method": "DELETE",
+                            "path": "/tasks/{id}"
+                        }
+                    ],
+                    "with":
+                    {
+                        "capability": "produce",
+                        "topic": "task-command",
+                        "key": "${params.id}",
+                        "reply-to": "task-query",
+                        "async":
+                        {
+                            "location": "/tasks/${params.id};${correlationId}"
+                        }
+                    }
+                }
+            ]
+        },
+        "sse0":
+        {
+            "type" : "sse",
+            "kind": "server",
+            "exit": "sse_kafka0"
+        },
+        "sse_kafka0":
+        {
+            "type" : "sse-kafka",
+            "kind": "proxy",
+            "routes":
+            [
+                {
+                    "exit": "kafka0",
+                    "when":
+                    [
+                        {
+                            "path": "/{topic}"
+                        }
+                    ],
+                    "with":
+                    {
+                        "topic": "${params.topic}"
+                    }
+                }
+            ]
+        },
+        "kafka0":
+        {
+            "type" : "kafka",
+            "kind": "cache_client",
+            "options":
+            {
+                "merged":
+                [
+                    "items-requests",
+                    "items-responses"
+                ],
+                "bootstrap":
+                [
+                    "items-responses"
+                ]
```

Comment:
These topics need to be changes to match with `task-commands` and `task-replies` topic names above.